READMEs/devops/deploy.md

```diff
@@ -0,0 +1,26 @@
+# Deploy Process
+
+Environment updates run through the following steps using GitHub Actions and ArgoCD:
```

Comment:
```suggestion
Environment updates run through the following steps using GitHub Actions and ArgoCD. These happen automatically once a commit is added to the `main` branch and passes the CI workflow.
```

Reply:
Doesn't this only run with a new production tag, not any passing commit to main?

Reply:
a passing commit to main triggers the new production tag, which triggers the rest of it

Reply:
Aha! I missed that bit in the workflow yaml.